src/components/pages/dashboard-pages/HomeStudent.jsx

```diff
@@ -73,29 +80,78 @@ const HomeStudent = ({ profile, onNavigate, profileError }) => {
           </div>
         ) : null}
       </div>
-      <div className="flex flex-col w-9/12 sm:w-7/12 gap-8 mt-4 self-center sm:self-start">
-        <div className="sm:h-2/5 flex flex-col bg-pureWhite">
+      <div className="flex flex-col w-9/12 sm:w-7/12 gap-2 mt-4 self-center sm:self-start h-full">
           <h2 className="font-spartan font-semibold text-2xl p-4">
             Your upcoming lessons
           </h2>
-          <div>
-            <p className="px-4">
-              {myLessons.length > 0 ? myClasses : 'No lessons booked yet'}
-            </p>
-          </div>
+          {nextTwoLessons.length > 0 ? (
+            <div className="flex gap-8 flex-col lg:flex-row">
+              {nextTwoLessons.map((lesson) => {
```

Comment:
lines 89-138 has a long return in map function. We could move this logic in a separate variable or even component.

Reply:
I'll think how to do this.